examples/resources/cloudavenue_catalog/resource.tf

```diff
@@ -0,0 +1,7 @@
+
+resource "cloudavenue_catalog" "test" {
```

Comment:
It may be convenient to fix a nomenclature of example naming. We have named all our firsts examples "example" instead of "test".